src/onboarding/welcome/Welcome.test.tsx

```diff
@@ -58,7 +58,7 @@ describe('Welcome', () => {
         acceptedTerms: true,
       },
     })
-    jest.mocked(firstOnboardingScreen).mockReturnValue(Screens.NameAndPicture)
+    jest.mocked(firstOnboardingScreen).mockReturnValue(Screens.ImportSelect)
```

Comment:
same as above, should this be PincodeSet?

Reply:
Fixed in 8767c90!